modin/engines/dask/pandas_on_dask/frame/partition.py

```diff
@@ -114,24 +90,34 @@ def apply(self, func, **kwargs):
         -----
         The keyword arguments are sent as a dictionary.
         """
-        func = pkl.dumps(func)
-        call_queue = self.call_queue + [[func, kwargs]]
-        future = default_client().submit(
-            apply_list_of_funcs, call_queue, self.future, pure=False
-        )
-        futures = [default_client().submit(lambda l: l[i], future) for i in range(2)]
+        client = default_client()
+        call_queue = self.call_queue + [[func, args, kwargs]]
```

Comment:
```suggestion
        call_queue = self.call_queue + [(func, args, kwargs)]
```
Let's use tuples where possible - they take less memory and could be slightly faster in certain operations.